client/resources/md5mesh.cpp

```diff
@@ -262,18 +262,19 @@ void MD5Mesh::CalculateVertexPositions() {
 	if (this->meshes.size() < this->meshes_internal.size()) {
 		this->meshes.reserve(this->meshes_internal.size());
 		for (std::size_t i = this->meshes.size(); i < this->meshes_internal.size(); ++i) {
-			CreateMesh();
+			CreateMesh()->has_weight = true;
```

Comment:
This seems like it should be a parameter, instead of having it return the created mesh and chain the property assignment off of it. What if CreateMesh throws or errors?

Reply:
if CreateMesh were to throw, that would be ideal, it would leave this whole function. if CreateMesh returned a null pointer, then the function would have dereferenced it anyways and would crash.